config/webpack.config.dev.js

```diff
@@ -102,12 +102,6 @@ module.exports = {
       'react-native': 'react-native-web',
     },
     plugins: [
-      // Prevents users from importing files from outside of src/ (or node_modules/).
```

Comment:
I removed this so we can import from outside the `src` folder to get the JSON contracts. It seemed cleaner then copying them into the `src` folder